build/repo.targets

```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <AutobahnReportDir>$(ArtifactsDir)autobahnreports\</AutobahnReportDir>
+  </PropertyGroup>
+
+  <Target Name="SetReportDirectory" BeforeTargets="Test">
```

Comment:
This would previously have been done right after `repo-initialize`. That's before the _`compile`_ target.